lib/test_transform.dart

```diff
@@ -91,15 +99,14 @@ class TestTransform implements Transform {
     if (package == id.package) {
       path = id.path;
     } else {
+      if (packageResolver == null) await _setupPackageResolver();
```

Comment:
Perhaps move the caching logic into an async getter?
```dart
Future<PackageResolver> get packageResolver() async {
  if (_packageResolver == null) {
    await _setupPackageResolver();
  }
  return _packageResolver;
}
```

Reply:
Ah, that's clearly better (and it gets to do what _setupPackageResolver used to do, there is no point in having that separately). Using a method, because getters are often assumed to "finish quickly".